__init__.py

```diff
@@ -75,3 +65,17 @@ def led_on():
 def led_off():
     GPIO.setup(LED_PIN, GPIO.OUT)
     GPIO.output(LED_PIN, GPIO.HIGH) # Turn off LED
+
+# def read_sensor_gas():
```

Comment:
Please remove the commented out code blocks. If the code will not be used, it should not be present in the file.
Since we have version control we can access it anywas if we need to in the future.